scripts/build-shotcut.sh

```diff
@@ -384,6 +386,17 @@ function set_globals {
     fi
   fi
 
+  if [ "$DEBUG_BUILD" = "1" ]; then
+    CONFIGURE_DEBUG_FLAG="--enable-debug"
```

Comment:
Consider initializing these variables so that we don't end up with uninitialized variables when the flag is not set. Same for ASAN below.